lib/options/dcu.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Documentation       Library for UEFI configuration using Dasharo Configuration
+...                 Utility tool. Commonly used when serial port is not
+...                 available.
+
+Library             Collections
+Library             OperatingSystem
+Library             String
+
+
+*** Keywords ***
+Set UEFI Option
+    [Documentation]    Set an UEFI option to a value.
+    [Arguments]    ${option_name}    ${value}
+    Run    git clone https://github.com/Dasharo/dcu
+    # TODO: Remove once smmstore support is merged
+    Run    cd dcu && git checkout smmstore > /dev/null 2>&1 && cd ..
+    Power On
+    Boot System Or From Connected Disk    ubuntu
```

Comment:
We assume Ubuntu is the first one in boot order here?
Do we have an idea for Windows tests?
In some suites we have a mixup of Ubuntu -> Windows -> Ubuntu, where tests must select which OS to boot over serial.

Reply:
Yeah on SSH this keyword is no-op, on Linux at least we can use efibootmgr to manipulate bootorder / bootnext, no idea about Windows though.

Reply:
Not sure what to do about this at this point. Right now novacustom only has `TESTS_IN_UBUNTU_SUPPORT` selected so it's a valid assumption that Ubuntu will always boot automatically

Reply:
https://github.com/Dasharo/open-source-firmware-validation/issues/313